package.json

```diff
@@ -1513,7 +1513,7 @@
         "createRelease": "ts-node ./build-scripts/createRelease.ts"
     },
     "devDependencies": {
-        "@aws-toolkits/telemetry": "0.0.80",
+        "@aws-toolkits/telemetry": "^0.0.91",
```

Comment:
I think we want to stay pinned (not `^`) so that old versions of the codebase are still buildable even if the telemetry package has breaking changes in the future. (Unlikely but good practice)

Reply:
Ah you're right. I'll have to change `package-lock.json` as well.